tests/modules/tfm/secure_services/src/main.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include <ztest.h>
+ #include <tfm_ns_interface.h>
+ #include <tfm/tfm_ioctl_api.h>
+ #include <pm_config.h>
+
+static bool is_secure(intptr_t ptr)
+{
+	/* We can not use 'arm_cmse_addr_is_secure here since this firmware
+	 * is built as non secure.
+	 */
+	if (ptr >= CONFIG_PM_SRAM_BASE) {
+		/* Check if RAM address is secure */
+		return ptr < PM_SRAM_ADDRESS;
+	}
+
+	/* Check if flash address is secure */
+	return ptr < PM_ADDRESS;
+}
+
+void test_tfm_read_service(void)
+{
+	const uint32_t ficr_start = (NRF_FICR_S_BASE + 0x204);
```

Comment:
On nRF53 NRF_FICR->INFO.CONFIGID is at 0x200.

Reply:
You are correct, I've raised a bug so it can be fixed at a later point